app/helper/multer.js

```diff
@@ -11,10 +11,8 @@ const storage = multer.diskStorage({
   },
 });
 
-const upload = multer({
-  storage: storage,
-});
+const upload = multer({ storage: storage }); // Modify here to use .single() method
```

Comment:
Also validate the file like which type of file will come and also check of how many size are required

Reply:
> Also validate the file like which type of file will come and also check of how many size are required

About this ?